kontent-ai/management-sdk-js
# Return workflow object view language variant

### Motivation

Which issue does this fix? Fixes #99 


### Checklist

- [✔ ] Code follows coding conventions held in this repo
- [✔ ] Automated tests have been added
- [✔] Tests are passing
- [✔ ] Docs have been updated (if applicable)
- [✔ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Use npm pack to create package tarball
Move tarball file to testing repo and use it to install the management SDK
Make view language variant request to live project and check response
